.github/ISSUE_TEMPLATE/gh-bug-report.yml

```diff
@@ -0,0 +1,108 @@
+name: Bug Report - GitHub Copilot integration
+description: File a bug report to help us improve GitHub Copilot integration for the MSSQL extension.
+title: "[GitHub Copilot - Bug]: "
+labels: ["bug", "PrPr", "Area - GitHub Copilot", "Triage: Needed"]
```

Comment:
Github added <Bug> as a top-level type, which is separate from (and IMO nicer than) having a <bug> label.  I don't think Karl's decided to standardize on that yet, but I've been trying to shift to that when filing and triaging submitted issues since it's a little nicer than using labels

Reply:
Yeah, I'm not sure if "Types" are available on these GitHub issue templates. We can always update the template but for now let's go with labels, we need this template in place ASAP to track bugs from the PrPr.